source/adapters/cuda/context.hpp

```diff
@@ -100,6 +100,21 @@ CreateHostMemoryProvider(ur_device_handle_t_ *DeviceHandle,
   return UR_RESULT_SUCCESS;
 }
 
+static ur_result_t
+CreateHostMemoryPool(umf_memory_provider_handle_t MemoryProviderHost,
```

Comment:
There is already a function to create proxy pool: https://github.com/oneapi-src/unified-runtime/blob/main/source/common/ur_pool_manager.hpp#L307

Reply:
Removed.